__tests__/integration.test.js

```diff
@@ -7,8 +7,19 @@ const app = require('../MVC/app')
 beforeEach(() => { return seed(testData) })
 afterAll(() => db.end())
 
-describe('GET/api/topics', () => {
-    test('check this runs', () => {
-
+describe('GET: /api/topics', () => {
```

Comment:
We can also test for any requests to a non-existent endpoint or set up error handling middleware too! :D